src/plugins/telemetryTable/TelemetryTableNameColumn.js

```diff
@@ -0,0 +1,44 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+define([
+    './TelemetryTableColumn.js'
+], function (
+    TelemetryTableColumn
+) {
+    class TelemetryTableNameColumn extends TelemetryTableColumn {
+        constructor(openmct, telemetryObject, metadatum) {
+            super(openmct, metadatum);
+
+            this.telemetryObject = telemetryObject;
+        }
+
+        getRawValue() {
+            return this.telemetryObject.name;
+        }
+
+        getFormattedValue() {
+            return this.telemetryObject.name;
+        }
+    }
+
+    return TelemetryTableNameColumn;
+});
```

Comment:
we dont need to write a new telemetry table column for this

Reply:
Actually this was instructed in issue `This can be done without modifying the telemetry data itself, by adding a TelemetryTableNameColumn that extends TelemetryTableColumn, and which overrides the getFormattedValue() method.`

Reply:
Understand the concern here. I recommended this approach because it's more performant than adding the name to the telemetry data (which we do in VISTA). It also neatly fits in to the current Table architecture.